### ISSUE TITLE
Cursor highlighting missing

### ISSUE THREAD

**Issue by jeremypw** [elementary/files#2501]
### What Happened?

Moving the view cursor with `<Ctrl>Arrow` has no visual feedback as to where the cursor is. 

### Steps to Reproduce

See above

### Expected Behavior

The position of the cursor should be discernible, independent of the selected status of the item.  This is the case in other filemanagers such as Nautilus and Dolphin.

### OS Version

8.x (Early Access)

### Session Type

Classic Session (X11, This is the default)

### Software Version

Latest release (I have run all updates)
